sass/main.scss

```diff
@@ -0,0 +1,177 @@
+*{
+  margin: 0px;
+  padding: 0px;
+}
+
+#topbar{
```

Comment:
Is it a good code practise to put a space between the named object and curly bracket.